packages/@aws-cdk/cloud-assembly-schema/schema/cloud-assembly.version.json

```diff
@@ -1 +1 @@
-{"version":"8.0.0"}
+{"version":"9.0.0"}
```

Comment:
maybe not a big deal, but shouldn't it be "8.1.0"?  Old CDK CLI refuses to deploy future major versions of cloud assembly, but technically this change doesn't prevent them from doing so

Reply:
Perhaps, but it will be very confusing for people otherwise if they have a framework version `>1.86.0` (say) and SUPPOSED to have this feature work, but it won't work because they're using a CLI `<1.86.0` (and nobody's telling them).

Better to force both upgrades to happen synchronously.

Reply:
> but it won't work because they're using a CLI 

If new bootstrap continues to create CF stack output old CLI is checking, then everything continues to work, no?

Reply:
That's not the issue. The issue is people depending on the "new feature" (advertised in the CHANGELOG) that you can now deploy even with a nonstandard bootstrap stack name--except it doesn't work! 

Why? Because they upgraded their framework but not their CLI.